playbooks/packstack.yaml

```diff
@@ -26,11 +29,7 @@
 
     - name: Install OpenStack {{ os_release }} repository
       yum: name=centos-release-openstack-{{ os_release }} state=installed
```

Comment:
Use the `var: value style`
Make sure to keep the values using variables ( {{ }} ) protected by interveted commas : 
`name=centos-release-openstack-{{ os_release }}` -> `name:  "centos-release-openstack-{{ os_release }}"`